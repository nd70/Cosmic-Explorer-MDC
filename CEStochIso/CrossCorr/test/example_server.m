stochastic_paths;
delete('example_server.out');
[success, msg, msgid] = rmdir('cachefiles_server', 's');
mkdir('cachefiles_server');
diary('example_server.out');
stochastic('example_server.txt', 'example_jobs.txt', 2);
diary off;
