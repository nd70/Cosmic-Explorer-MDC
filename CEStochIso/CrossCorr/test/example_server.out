segment=3, trial =1, start sec = 1126627369, CC stat = 3.070804e-04, theor sigma = 4.385573e-04
segment=4, trial =1, start sec = 1126627399, CC stat = -4.721304e-05, theor sigma = 4.331303e-04
segment=5, trial =1, start sec = 1126627429, CC stat = 3.808077e-04, theor sigma = 4.276659e-04
segment=6, trial =1, start sec = 1126627459, CC stat = 6.348086e-04, theor sigma = 4.308190e-04
segment=7, trial =1, start sec = 1126627489, CC stat = -2.095123e-04, theor sigma = 4.289766e-04
segment=8, trial =1, start sec = 1126627519, CC stat = 3.236710e-04, theor sigma = 4.204094e-04
segment=9, trial =1, start sec = 1126627549, CC stat = 3.416989e-04, theor sigma = 4.170852e-04
segment=10, trial =1, start sec = 1126627579, CC stat = 2.491606e-04, theor sigma = 4.275095e-04
segment=11, trial =1, start sec = 1126627609, CC stat = 1.435865e-04, theor sigma = 4.425026e-04
segment=12, trial =1, start sec = 1126627639, CC stat = -4.610263e-04, theor sigma = 4.603682e-04
segment=13, trial =1, start sec = 1126627669, CC stat = -7.082224e-04, theor sigma = 4.507563e-04
segment=14, trial =1, start sec = 1126627699, CC stat = 1.889514e-04, theor sigma = 4.548161e-04
segment=15, trial =1, start sec = 1126627729, CC stat = -1.111019e-04, theor sigma = 4.899160e-04
segment=16, trial =1, start sec = 1126627759, CC stat = 1.848009e-04, theor sigma = 5.036686e-04
segment=17, trial =1, start sec = 1126627789, CC stat = -6.454592e-04, theor sigma = 4.764881e-04
segment=18, trial =1, start sec = 1126627819, CC stat = 1.081851e-03, theor sigma = 4.761411e-04
segment=19, trial =1, start sec = 1126627849, CC stat = -4.415706e-04, theor sigma = 4.827310e-04
segment=20, trial =1, start sec = 1126627879, CC stat = -5.005905e-04, theor sigma = 4.671271e-04
segment=21, trial =1, start sec = 1126627909, CC stat = 4.480586e-04, theor sigma = 4.535013e-04
segment=22, trial =1, start sec = 1126627939, CC stat = 2.599749e-04, theor sigma = 4.543806e-04
segment=23, trial =1, start sec = 1126627969, CC stat = -4.062091e-04, theor sigma = 4.388201e-04
